pkg/ipamd/rpc_handler.go

```diff
@@ -198,16 +198,11 @@ func (s *server) DelNetwork(ctx context.Context, in *rpc.DelNetworkRequest) (*rp
 	eni, ip, deviceNumber, mismatchedStore, err := s.ipamContext.dataStore.UnassignPodIPv4Address(ipamKey)
```

Comment:
As discussed, we can do away with `mismatchedStore` and instead rely on `addr.Prefix.IP`value to determine if it's a secondary IP or Prefix.

Reply:
Yeah, now it won't be needed. Will remove it.

Reply:
Yes..